src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/index.jsx

```diff
@@ -3,25 +3,19 @@ import { useHistory } from 'react-router-dom';
 import { connect } from 'react-redux';
 import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
 import FormButtons from '../../../components/FormButtons';
-import { LANGUAGES } from '../../../utils/constants';
 import * as actions from '../../redux/actions';
 import { getFormPageInfo } from '../../../utils/selectors';
 import { scrollAndFocus } from '../../../utils/scrollAndFocus';
 import ProviderSelectionField from './ProviderSelectionField';
 
 const initialSchema = {
   type: 'object',
-  required: ['preferredLanguage'],
+  required: [],
```

Comment:
should be able to just leave this property off now

Reply:
The reducer conditionally pushes a field to this array so I left it blank, but will remove this and update the reducer instead.